src/main/java/io/hhplus/tdd/point/PointService.java

```diff
@@ -0,0 +1,19 @@
+package io.hhplus.tdd.point;
+
+import io.hhplus.tdd.database.UserPointTable;
+import org.springframework.stereotype.Service;
+
+@Service
+public class PointService {
+    private final UserPointTable userPointTable;
+
+    public PointService(UserPointTable userPointTable) {
+        this.userPointTable = userPointTable;
+    }
+
+    public UserPoint chargePoint(long id, long amount) {
+        UserPoint userPoint = userPointTable.selectById(id);
```

Comment:
유저가 없는 경우에 대해서도 예외가 추가 되면 좋을것 같습니다~
```suggestion
        Optional<UserPoint> userPoint = Optional.ofNullable(userPointTable.selectById(id));
```